docs/decisions/template.md

```diff
@@ -0,0 +1,80 @@
+---
+# These are optional elements. Feel free to remove any of them.
+status: {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)}
+date: {YYYY-MM-DD when the decision was last updated}
+deciders: {list everyone involved in the decision}
+consulted: {list everyone whose opinions are sought (typically subject-matter experts); and with whom there is a two-way communication}
+informed: {list everyone who is kept up-to-date on progress; and with whom there is a one-way communication}
```

Comment:
I would skip `consulted` and specially `informed`. I don't think it adds much value. If anything special related to this happens, we can mention it in the relevant section (consulted in decision drivers,  informed in more info)